pkg/kyma/docs/install.md

```diff
@@ -0,0 +1,79 @@
+# install
+
+## Description
+
+Installs Kyma on a running Kubernetes cluster.
+
+## Usage
+
+```
+kyma install [OPTIONS]
+```
+
+## Options
+
+| Name     | Short Name | Default value| Description|
+| ----------|---------|-----|------|
+| --release | -r ||Kyma release to use|
```

Comment:
please mention where to find it. Furthermore you can use any git revision here like 'master', not only real rease versions